foxglove/console/lib.go

```diff
@@ -145,14 +145,18 @@ func Login(ctx context.Context, client *FoxgloveClient, authDelegate AuthDelegat
 	if err != nil {
 		return "", fmt.Errorf("failed to fetch device code: %w", err)
 	}
-	fmt.Println("Enter this code in your browser: ", info.UserCode)
 	browser, err := authDelegate.openBrowser(info.VerificationUriComplete)
-	if err != nil {
-		return "", fmt.Errorf("failed to open browser: %w", err)
+	if err == nil {
+		fmt.Println("Enter this code in your browser: ", info.UserCode)
+		defer func() {
+			_ = browser.Process.Kill()
+		}()
+	} else {
+		fmt.Printf("Failed to open browser window (%s)\n", err)
+		fmt.Println("Copy/paste the following link into your browser and enter this code: ", info.UserCode)
```

Comment:
The wording here is a little confusing, which is because the existing wording is too. AFAIK, the user is never prompted to enter a code. So maybe:

```suggestion
		fmt.Println("Copy/paste the following link into your browser: ", info.UserCode)
```

And maybe the above (opened browser) case to something like:

> Verify this code in your open browser: